com.unity.render-pipelines.high-definition/Runtime/Material/LayeredLit/LayeredLit.shader

```diff
@@ -413,7 +413,7 @@ Shader "HDRP/LayeredLit"
     // _ENABLESPECULAROCCLUSION keyword is obsolete but keep here for compatibility. Do not used
     // _ENABLESPECULAROCCLUSION and _SPECULAR_OCCLUSION_X can't exist at the same time (the new _SPECULAR_OCCLUSION replace it)
     // When _ENABLESPECULAROCCLUSION is found we define _SPECULAR_OCCLUSION_X so new code to work
-    #pragma shader_feature _ENABLESPECULAROCCLUSION                     // Non-local 
```

Comment:
Looks like this slipped in because my local diff didn't show whitespace. Hopefully it's still ok.